Index: app/src/main/java/com/example/pabs/Models/Event.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.pabs.Models;\r\n\r\nimport android.net.Uri;\r\n\r\n/**\r\n * Event cards\r\n */\r\n\r\npublic class Event {\r\n    private String Title;\r\n    private Uri Thumbnail;\r\n    private String StartDate;\r\n\r\n    public Event(){}\r\n\r\n    public Event(String title, Uri thumbnail, String startDate) {\r\n        Title = title;\r\n        Thumbnail = thumbnail;\r\n        StartDate = startDate;\r\n    }\r\n\r\n    public String getTitle() {\r\n        return Title;\r\n    }\r\n\r\n    public void setTitle(String title) {\r\n        Title = title;\r\n    }\r\n\r\n    public Uri getThumbnail() {\r\n        return Thumbnail;\r\n    }\r\n\r\n    public void setThumbnail(Uri thumbnail) {\r\n        Thumbnail = thumbnail;\r\n    }\r\n\r\n    public String getStartDate() { return StartDate; }\r\n\r\n    public void setStartDate(String startDate) { StartDate = startDate; }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/pabs/Models/Event.java	(revision e9d66e5c911efdc6d137cae53bb23c15186bb5ed)
+++ app/src/main/java/com/example/pabs/Models/Event.java	(date 1608131745846)
@@ -9,14 +9,12 @@
 public class Event {
     private String Title;
     private Uri Thumbnail;
-    private String StartDate;
 
     public Event(){}
 
     public Event(String title, Uri thumbnail, String startDate) {
         Title = title;
         Thumbnail = thumbnail;
-        StartDate = startDate;
     }
 
     public String getTitle() {
@@ -34,8 +32,4 @@
     public void setThumbnail(Uri thumbnail) {
         Thumbnail = thumbnail;
     }
-
-    public String getStartDate() { return StartDate; }
-
-    public void setStartDate(String startDate) { StartDate = startDate; }
 }
