Index: app/src/main/java/com/example/pabs/Models/Task.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/pabs/Models/Task.java	(date 1608587083010)
+++ app/src/main/java/com/example/pabs/Models/Task.java	(date 1608587083010)
@@ -0,0 +1,52 @@
+package com.example.pabs.Models;
+
+import sun.bob.mcalendarview.vo.DateData;
+
+public class Task {
+    public String origin;
+    public String description;
+    public DateData deadline;
+    public boolean done;
+
+    public Task(){
+    }
+
+    public Task(String origin, String description, DateData deadline, boolean done) {
+        this.origin = origin;
+        this.description = description;
+        this.deadline = deadline;
+        this.done = done;
+    }
+
+    public String getOrigin() {
+        return origin;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public DateData getDeadline() {
+        return deadline;
+    }
+
+    public boolean isDone() {
+        return done;
+    }
+
+    public void setOrigin(String origin) {
+        this.origin = origin;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setDeadline(DateData deadline) {
+        this.deadline = deadline;
+    }
+
+    public void setDone(boolean done) {
+        this.done = done;
+    }
+}
